src/widgets/DataTable/HeaderRow.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/forbid-prop-types */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, StyleSheet } from 'react-native';
+
+const HeaderRow = React.memo(({ renderCells }) => (
```

Comment:
Should we do doc strings for all these components now? Dunno about making an issue for that or probably forget?

Reply:
Do now I think